### PR TITLE
Register derived from (resolve)

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -88,21 +103,46 @@ fn run() -> Result<()> {
...
+    // * Allow users the option of setting complex logging filters using
+    //   env_logger's `RUST_LOG` environment variable.
+    // * Override both of those if the logging level is set via the `--log`
+    //   command line argument.
+    let env = env_logger::Env::default()
+        .filter_or(env_logger::DEFAULT_FILTER_ENV, "info");
+    let mut builder = env_logger::Builder::from_env(env);
+    builder.default_format_timestamp(false);
+
+    let log_lvl_from_env =
```
Comment (therealprof) [inline src/main.rs]: This looks shady. Why store it in a variable and then ignore it?

Comment (burrbull) [reply src/main.rs]: I have no idea. All commits except mine are already in master.

This PR is only due to:
https://github.com/rust-embedded/svd2rust/pull/308#issuecomment-511140315

**Comment by therealprof** [comment]
> I have no idea. All commits except mine are already in master.

Would you mind rebasing/squashing this then?

**Comment by therealprof** [comment]
NB: I'm not quite sure how bors handles PRs to non-master branches.

**Comment by burrbull** [comment]
> > I have no idea. All commits except mine are already in master.
> 
> Would you mind rebasing/squashing this then?

I don't know how to rebase and not lose authorship. Maybe you can do this?

**Comment by burrbull** [comment]
Or I can reopen #308 .

**Comment by therealprof** [comment]
Hm, okay. On the off-chance messing everything up, I'm going to try something. 😅

**Comment by therealprof** [comment]
Yay! I just created more conflicts. 💯

**Pull request merged**